lib/cli/index.js

```diff
@@ -32,6 +32,7 @@ exports.run = () => {
         .allowUnknownOption(true)
         .option('-r, --reporter <reporter>', 'test result reporter (flat by default)', collect)
         .option('--html-reporter-path <path>', 'relative path where html reporters will be stored', collect)
+        .option('--html-failed-only', 'html report generates only for failed tests')
```

Comment:
To my mind we should look wider. And create option that not only responsible for generating a report with failed tests, but can be configurable. For example if someone want generate report only with passed tests or with skipped and retried tests. I'm not sure that it is really common case.

But in my opinion everything that can be configured must be configured.

So I suggest to call this option as a `--html-reporter-only <state>`.

@sipayRT what do you think about my proposal?